ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue

```
@@ -130,11 +132,19 @@ export default defineComponent({
       labels
     } = useAddress(toRefs(props).value, addressSchema)
 
-    const { addressForm, resetValidation, validate } = useBaseValidations()
-
     const { enableAddressComplete, uniqueIds } = useAddressComplete(addressLocal)
 
-    const isValidCivicAddress = false
+    const localState = reactive({
+      isValidCivicAddress: false,
+      addressLocal
+    })
+
+    const validateForm = (): void => {
+      if (props.validate) {
+        // @ts-ignore - function exists
+        if (context.$refs.street.valid && context.$refs.city.valid) localState.isValidCivicAddress = true
```

Could this be simplified to `localState.isValidCivicAddress = context.$refs.street.valid && context.$refs.city.valid` ?

---

In changing to your suggestion. I found another issue, regarding the context, so fixed that.  I did some thorough testing. I will be testing more on my next ticket, but validation seems to catch all exceptions..